eZ/Publish/Core/Base/Exceptions/ContentFieldValidationException.php

```diff
@@ -30,17 +31,27 @@ class ContentFieldValidationException extends APIContentFieldValidationException
      */
     protected $errors;
 
+    /** @var string|null */
+    protected $contentName;
+
     /**
      * Generates: Content fields did not validate.
      *
      * Also sets the given $fieldErrors to the internal property, retrievable by getFieldErrors()
      *
      * @param array<array-key, array<string, \eZ\Publish\Core\FieldType\ValidationError>> $errors
      */
-    public function __construct(array $errors)
+    public function __construct(array $errors, ?string $contentName = null)
```

Comment:
Probably static factory method might be better choice to express your intention: creating ContentFieldValidationException for content update scenario

Reply:
I second that. A new static method to create a new instance with multiline message would be a cleaner approach.